### PR TITLE
Moar packages

### PR DESCRIPTION
**Pull request by ivoflipse**
A merge conflict? I even bothered to rebase on the tip :(

### CODE REVIEW

File: aniso8601/meta.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 package:
-  name: sphinx_rtd_theme
-  version: "0.1.8"
+  name: aniso8601
```
Comment (msarahan) [inline aniso8601/meta.yaml]: Did you really mean to replace the sphinx_rtd_theme package here?  I don't think aniso is a replacement.

**Comment by msarahan** [comment]
Sorry, I can't merge this as is. You have several accidental package renames/removals. It would be nice if you broke this up into smaller chunks, also.

These are valuable recipes, though.  Ping me if you need help resubmitting.

**Comment by ivoflipse** [comment]
Woops, I accidentally made a PR against this repo, instead of our own fork...

So I'll withdraw the PR and see if I can add more piecemeal packages at some other point

**Comment by msarahan** [comment]
No problem - thanks.

**Pull request closed without merging**